Formula/seqan@3.rb

```diff
@@ -15,15 +15,25 @@ class SeqanAT3 < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "doxygen" => :build
   depends_on "xz" => :build
+  depends_on "gcc@8"
 
-  def install
-    system "cmake", "test/documentation/"
-    system "make", "doc_usr"
+  # requires c++17 and concepts
+  fails_with :clang do
+    cause "seqan3 requires concepts and c++17 support"
+  end
 
-    include.install "include/seqan3"
-    doc.install Dir["#{buildpath}/doc_usr/html/*"]
+  ["4.9", "5", "6"].each do |n|
```

Comment:
Ok, just being curious here but is there a more elegant way to blacklist the compilers. Like all compilers below a certain version?
There does not seem to be a specific comment for this, is there? But would it be at least possible to iterate over all available compilers, extract their version number, and check if it is above a certain major version?
Haven't seen anything like this in the other Formulas.

Reply:
No, sorry, I would usually use…
```ruby
fails_with gcc: "4.9" # requires C++17
fails_with gcc: "5" # requires C++17
fails_with gcc: "6" # requires C++17
```

Reply:
I'd prefer to unroll the loop.